javascript/ql/test/query-tests/Security/CWE-400/PrototypePollution.ql

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name Prototype Pollution
+ * @description Recursively merging a user-controlled object into another object
+ *              can allow an attacker to modify the built-in Object prototype. 
+ * @kind path-problem
+ * @problem.severity warning
+ * @precision high
```

Comment:
This should perhaps be `low`, considering that we don't check package versions?

Reply:
And in fact if you lower its precision we can move that query into `ql/src`, since it won't be run on LGTM that way.